docs/marketplace/add_to_marketplace/manage_marketplace_items.md

```diff
@@ -0,0 +1,117 @@
+---
+id: manage_marketplace_items
+title:  Manage Marketplace Items
+sidebar_label:  Manage Marketplace Items
+---
+
+There are three main methods to create, modify and delete Marketplace items:
+
+* (recommended) Use [`miactl`](/cli/miactl/10_overview.md), the Mia-Platform command line interface tool.
+* Open an issue on Mia-Platform [Github community page](https://github.com/mia-platform/community).
+* (deprecated) Use the [CMS](/business_suite/guide_cms.md).
+
+## Use `miactl`
+
+
+First of all, you need to setup `miactl`, as explained in the [dedicated doc](/cli/miactl/20_setup.md).
+
+With the `miactl marketplace` subcommands, you can perform several actions, described here below.
+
+### Create an item 
+
+:::info
+
+You need to have *Company Owner* or *Project Administrator* permission to perform this action
+
+:::
+
+First of all, you need to create a JSON file as explained in [this guide](/marketplace/add_to_marketplace/contributing_overview.md#how-to-configure-a-new-component). 
+Save the file in the current directory, for example as `myMarketplaceItem.json` file.
+
+
+
+To create the item on the Marketplace, simply run this command:
+
+```sh
+miactl marketplace apply -f myMarketplaceItem.json
+```
+
+A message will confirm the operation, returning some information:
+```
+1 of 1 items have been successfully applied:
+
+  ITEM ID                   NAME        STATUS   
+
+  65368hf0c91d871a87afbcbf  API Portal  Inserted  
+```
+
+
+If your object contains the `image` and/or the `supportedByImage` keys, such images will be uploaded along with the item.
+The keys will be replaced with the `imageUrl` and the `supportedByImageUrl`, to obtain the updated version of the item use the `get` command: 
+```sh
+miactl marketplace get ITEM_ID > myMarketplaceItem.json
+```
```

Comment:
We can provide a picture of the item correctly created inside the Marketplace of the Console